daisy_workflows/image_import/import_image.sh

```diff
@@ -139,14 +139,17 @@ fi
 
 # Ensure the output disk has sufficient space to accept the disk image.
 # Disk image size info.
-SIZE_BYTES=$(qemu-img info --output "json" "${IMAGE_PATH}" | grep -m1 "virtual-size" | grep -o '[0-9]\+')
+FILE_INFO=$(qemu-img info --output "json" "${IMAGE_PATH}")
+SIZE_BYTES=$(echo $FILE_INFO | grep -oP  '(?<="virtual-size":[ *])[0-9]+' | head -1)
```

Comment:
Is [jq](https://stedolan.github.io/jq/) available on the worker? If not, probably a good idea to add it and use it here.

Reply:
It's not. In order to use it, we need to add it to debian-9-worker. I will ask Zach later whether it's proper to add this one more tool (in other words, dependency). It's enough for now to use regex to deliver this specific feature.

Reply:
> It's enough for now to use regex to deliver this specific feature.

If it were enough than I wouldn't have given the feedback before :)

Bash commands are fragile and difficult to debug. Let's get jq added (or something similar); doing that will also address part of Zoran's concerns below.

Reply:
Adding an extra dependency is relatively harder (we need to release a new debian-9-worker before we release this change). I won't add if I can find a as-good solution. So I tried a different way.

In previous PR, I used json just because it's already there. We can actually use regular output, which is super easy and stable to parse. It looks like:

tzz@tzz:~/t$ qemu-img info gce_export_new
image: gce_export_new
file format: raw
virtual size: 17M (17350656 bytes)
disk size: 17M

So I just grep "file format" and get the content after comma. It just matches a fixed and simple single-line format, so it would be safe enough.